tool/src/java/org/sakaiproject/lessonbuildertool/tool/beans/SimplePageBean.java

```diff
@@ -7346,4 +7354,24 @@ public static String utf8truncate(String input, int length) {
 	return result.toString();
     }
 
+	/**
+	 * Method to add Resources folder into Lessons tool
+	 */
+	public String folderPickerSubmit(){
+		//Check if user has submitted page without selecting folder?
+		String defaultPath = "/group/" + getCurrentSiteId();
+		if(folderPath == null ||  folderPath.equals(defaultPath)){
+			return "failure";
+		}
+		String dataDirectory = defaultPath + "/" + folderPath;
+		contents = "<div data-copyright=\"true\" data-description=\"true\" data-directory=" +dataDirectory+ " data-files=\"true\" data-folder-listing=\"true\"></div>";
+		Placement placement = toolManager.getCurrentPlacement();
+		//setting filter false to disable antisamy for folder listing
+		placement.getPlacementConfig().setProperty("filterHtml","false");
```

Comment:
moved the javascript code from folder-picker.html to  folder-picker.js file and have pushed the changes.